readthedocs/templates/core/project_detail_right.html

```diff
@@ -61,16 +61,21 @@ <h3>{% trans "Badge" %}</h3>
 
 
 {% block tags %}
-  {% if project.tags.count %}
     <h3>{% trans "Tags" %}</h3>
     <p>
+  {% if project.tags.count %}
       {% for tag in project.tags.all %}
         <a href="{% url "projects_tag_detail" tag.slug %}">{{ tag.name }}</a>{% if forloop.last %}{% else %}, {% endif %}
       {% empty %}
         <span class="quiet">{% trans "No tags" %}</span>
```

Comment:
This condition won't be met since it has to be 0 on `project.tags.count` to have an empty list. I think you can remove the empty block.